docs/octant-plugin-installation.md

```diff
@@ -97,7 +97,7 @@ based on your environment and move the binary to OCTANT_PLUGIN_PATH.
     For example, you can get antrea-octant-plugin-linux-x86_64 if it matches your operating system and architecture.
 
     ```bash
-    wget -O antrea-octant-plugin https://github.com/vmware-tanzu/antrea/releases/download/v0.8.1/antrea-octant-plugin-linux-x86_64
+    wget -O antrea-octant-plugin https://github.com/vmware-tanzu/antrea/releases/download/$VERSION/antrea-octant-plugin-linux-x86_64
```

Comment:
use `<TAG>` instead of `$VERSION` to be consistent with the rest of the document and with getting-started.md

`$VERSION` kind of implies that there is an env variable defined at some point